bin/rose-date

```diff
@@ -175,4 +173,31 @@
 #
 #     For example, for a duration `P57DT12H` - `y,m,d,h` -> `0,0,57,12`
 #-------------------------------------------------------------------------------
-exec python3 -m rose.date "$@"
+# -*- coding: utf-8 -*-
+import re
+import sys
+import os
+
+from isodatetime.main import main
+
+if __name__ == '__main__':
+    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
```

Comment:
Could you explain the purpose of this to me. I'm a bit confused as to why we are dealing with `.exe` extensions.

Reply:
No. I have no idea what it is or where it came from. Removing it.